src/Compilers/Core/AnalyzerDriver/AnalyzerDriverHelper.cs

```diff
@@ -13,76 +13,63 @@ namespace Microsoft.CodeAnalysis.Diagnostics
 {
     internal class AnalyzerDriverHelper
     {
-        internal const string DiagnosticId = "AD0001";
+        private const string DiagnosticId = "AD0001";
         private const string DiagnosticCategory = "Compiler";
```

Comment:
just curious. what makes something to go into this AnalyzerDriverHelper? some of them seems part of AnalyzerDriver and some other seems not related to each other. so just asking.